### DISCUSSION TITLE
`yarn native` doesn't work with Solito starter

### DISCUSSION THREAD

**Discussion by albertcito** [nandorojo/solito#420] [Q&A] [answered]
I have node 18 and I ran this command:

`npx create-solito-app@latest -t with-expo-router`
.....
`yarn native`

It displays this message:

> Development build: Unable to get the default URI scheme for the project. Please make sure the expo-dev-client package is installed.

And a QR code to go to this URL: 

> Metro waiting on http://localhost:8081

I don't know if it's an issue or if it's only my computer. How I can solve it please?

**Comment by derek-primumco** [comment, marked as answer]
See the release notes to version 3.1.1 at https://github.com/nandorojo/solito/releases/
<img width="740" alt="image" src="https://github.com/nandorojo/solito/assets/122907173/ed27c6ab-1678-4101-a42c-d149ad3692b9">
